editor/editor_help.h

```diff
@@ -195,10 +197,19 @@ class EditorHelp : public VBoxContainer {
 	static String doc_version_hash;
 	static Thread worker_thread;
 
+	static bool script_docs_loaded;
```

Comment:
```suggestion
	inline static bool script_docs_loaded = false;
```
You can do the same with other properties.